tests/integration/test_experiments.py

```diff
@@ -32,3 +35,43 @@ def test_ensembles(client, create_experiment, create_ensemble):
     # The list of ensembles belonging to the newly created experiment matches
     resp = client.get_check(f"/experiments/{experiment_id}/ensembles")
     assert ids == {ens["id"] for ens in resp.json()}
+
+
+def test_delete_experiment(client, create_experiment, create_ensemble):
+    experiment_id = create_experiment("1")
+
+    ensemble_id = create_ensemble(experiment_id, ["param1", "param2"])
+    matrix = np.random.rand(5, 8, 13)
+
+    # POST
+    post_url = f"/ensembles/{ensemble_id}/records/mat/matrix"
+    resp = client.post_check(post_url, json=matrix.tolist())
+    OBSERVATIONS = {
+        "OBS1": {
+            "values": [1, 2, 3],
+            "errors": [0.1, 0.2, 0.3],
+            "x_axis": ["0", "1", "2"],
+        },
+        "OBS2": {
+            "values": [2, 4, 3],
+            "errors": [0.9, 0.2, 0.3],
+            "x_axis": ["2", "3", "4"],
+        },
+        "OBS3": {
+            "values": [1, 4, 5],
+            "errors": [0.1, 0.5, 0.2],
+            "x_axis": ["5", "6", "7"],
+        },
+    }
+    for name, obs in OBSERVATIONS.items():
+        client.post_check(
+            f"/experiments/{experiment_id}/observations",
+            json=dict(
+                name=name,
+                values=obs["values"],
+                errors=obs["errors"],
+                x_axis=obs["x_axis"],
+            ),
+        )
+    resp = client.post_check(f"/experiments/{experiment_id}/delete")
+    assert len(client.get_check(f"/experiments").json()) == 0
```

Comment:
Also test that the observations that have been created have been deleted.

Reply:
Ie, I would expect a 404 if I try to access the observations tied to the experiment.

Reply:
Done!